examples/jolokia-wildfly-example/Makefile

```diff
@@ -1,21 +1,15 @@
 DOCKER_NAME = hawkular/hawkular-openshift-agent-example-jolokia-wildfly
-DOCKER_VERSION ?= dev
+DOCKER_VERSION ?= latest
 DOCKER_TAG = ${DOCKER_NAME}:${DOCKER_VERSION}
 
 all: build
 
 build:
-	if [ ! -f "jolokia-war-1.3.5.war" ]; then \
-		wget http://repo1.maven.org/maven2/org/jolokia/jolokia-war/1.3.5/jolokia-war-1.3.5.war ; \
-	fi
-	if [ ! -f "wildfly-10.1.0.Final.tar.gz" ]; then \
-		wget http://download.jboss.org/wildfly/10.1.0.Final/wildfly-10.1.0.Final.tar.gz ; \
-	fi
 	docker build -t ${DOCKER_TAG} .
 
 openshift-deploy: openshift-undeploy
 	@echo Deploying the Jolokia WildFly Example to OpenShift. Using project `oc project --short`
-	oc process -f jolokia-wildfly.yaml | oc create -f -
+	oc process -f jolokia-wildfly.yaml -v IMAGE_VERSION=latest | oc create -f -
```

Comment:
shouldn't this be "-v IMAGE_VERSION=${DOCKER_VERSION}" ... the default value of that variable is already 'latest' but this would allow someone to change what they want to deploy by:

```
DOCKER_VERSION=dev make openshift-deploy
```